### PR TITLE
Create abnormalBiologicalProcessBeforeStage.yaml

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes https://github.com/obophenotype/upheno/issues/734

related to https://github.com/obophenotype/upheno/pull/736
This PR failed, therefore this one probably will too. I should fix this pattern once I know what the fix is for #736

### CODE REVIEW

**Comment by Clare72** [comment]
@nicolevasilevsky - this just seems to be a timeout issue.

I can successfully run `sh run.sh make pattern_ontology` on this branch locally, but travis times out after 10 min, and that step takes my machine ~15 min.

You might be able to fix this by using travis_wait in your .travis.yml, similar to this:
https://github.com/FlyBase/drosophila-phenotype-ontology/blob/master/.travis.yml

or turning off imports, but I am not sure if this is possible on this repo.

File: src/patterns/dosdp-dev/abnormalBiologicalProcessBeforeStage.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+pattern_name: abnormalBiologicalProcessBeforeStage
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalBiologicalProcessBeforeStage.yaml
+description: "An abnormal biological process before a developmental stage in an organism."
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormalBiologicalProcessBeforeStage.yaml:3]: Can we get a slightly better description, in particular clarifying the term before and adding an example?

**Review by matentzn — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
🎂

File: src/patterns/dosdp-dev/abnormalBiologicalProcessBeforeStage.yaml (hunk 1 above)
Comment (nicolevasilevsky) [reply src/patterns/dosdp-dev/abnormalBiologicalProcessBeforeStage.yaml]: I added a better desc.

**Pull request merged**